src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      if (a === 0) {
+        return this;
+      } else {
+        return a / b;
+      };
+    },
+    reset() {
+      this.result = 0;
```

Comment:
Add empty line between methods 
```suggestion
    add(a, b) {
      return a + b;
    },
    
    subtract(a, b) {
      return a - b;
    },
    
    multiply(a, b) {
      return a * b;
    },
    
    divide(a, b) {
      if (a === 0) {
        return this;
      } else {
        return a / b;
      };
    },
    
    reset() {
      this.result = 0;
```